docs/data/getting-started/overview/overview.mdx

```diff
@@ -9,4 +9,4 @@ description: Get started with Pigment CSS, a zero-runtime CSS-in-JS library.
 
 ## Installation
 
-Pigment CSS has two category of packages. First one is to be imported and used in your source code. This includes the public API of the package. Second category is to be imported in your bundler config file to configure and actually be able to use Pigment CSS. We currently support [`Next.js`](https://nextjs.org/) (no Turbopack yet), [`Vite`](https://vite.dev/) and [`Webpack`](https://webpack.js.org/).
```

Comment:
These are just stubs in the docs right now.

Reply:
Hopefully, it will make upgrading the monorepo dependency easier 😄